### ISSUE TITLE
Missing merge lines command

### ISSUE THREAD

**Issue by ifaniqbal** [emmetio/sublime-text-plugin#83]
I cannot execute merge lines command in the recent update.

**Comment by adragin** [comment]
But ST3 has a built-in command (CTRL+J) for it, has it?

**Comment by ifaniqbal** [comment]
Never use that. I use the one from command palette and it was from Emmet.
I tried CTRL+J to join html tags but it doesn't work like Emmet Merge Lines

**Comment by danielhaim1** [comment]
The join lines command on a Mac: ⌘+J

**Comment by rchl** [comment]
Built-in join won't handle many cases properly. For example joining attributes won't add a space between attributes:
```html
<div
  attr1="a"
  attr2="b"
```
becomes
```html
<div
  att1="a"attr2="b"
```

**Comment by danielhaim1** [comment]
I too, am in agony from this change @rchl ⚰️

**Comment by lapinvert** [comment]
Please implement merge_line command back!

**Comment by adragin** [comment]
But the ST3 built-in command works just like that. It concatenates lines, removes excess whitespace (tab, some spaces), and adds only one space between the lines that are concatenated. And it doesn't depend on the file type.

**Comment by rchl** [comment]
@adragin nope: https://github.com/emmetio/sublime-text-plugin/issues/83#issuecomment-687830636
